code/game/objects/items/religion.dm

```diff
@@ -72,6 +72,34 @@
 /obj/item/banner/proc/special_inspiration(mob/living/carbon/human/H) //Any banner-specific inspiration effects go here
 	return
 
+/obj/item/banner/greytide
+	name = "greytopia banner"
+	desc = "The banner of the greytide, known to inspire fear in even the most hardened crew."
```

Comment:
Do we really, really want to encourage the "being a shitter" aspect of playing assistant? If this was PR'd on its own, I would close that PR. Which is probably why it's part of a ten thousand line PR

Reply:
Greytide is part of this games ---DEEP LORE---. Some rare item that gives a modest buff to the holder (they don't even have to be an assistant to use it) should not be that offensive to you. 

This was literally the entire inspiration for the PR, adding a chance for an assistant-themed room to appear, and it was one of the first items added. The actual greytide has been dead for years, don't be one of those shitposters who acts like an IC-reference to part of this game's history is going to bring back 2013 greytiding.

Reply:
can't stop the tide, it's part of the game now, might as well embrace it in the code